docs/api.md

```diff
@@ -3538,6 +3539,9 @@ Whether this request is driving frame's navigation.
 #### request.postData()
 - returns: <?[string]> Request's post body, if any.
 
+#### request.postDataJSON()
+- returns: <?[Object]> Parsed request's body if the request is JSON or form-urlencoded if any.
```

Comment:
Let's expand below on the logic - I missed the `form-urlencoded` bit in the docs when I was reading it for the first time.

Reply:
Feel free to adjust the wording, not sure if it's clear now.